dockertest/environment.py

```diff
@@ -62,3 +62,24 @@ def get_selinux_context(path):
     """
     # First list item is null-terminated string length
     return selinux.getfilecon(path)[1]
+
+
+def selinux_is_enforcing():
+    """
+    Return True if selinux is in enforcing mode.
+    """
+    # These constants are undefined in selinux module
+    # pylint: disable=C0103
+    ERROR = -1
+    OLDKERNEL = 0
+    ENFORCING = 1  # Travis complains about missing selinux.ENFORCING
+    try:
+        if selinux.selinux_status_open(OLDKERNEL) == ERROR:
+            raise ValueError("Could not r/o mmap /sys/fs/selinux/status")
+        mode = selinux.selinux_status_getenforce()
+        if mode == ERROR:
+            raise ValueError("Error checking SELinux is disabled,"
+                             " permissive, or enforcing")
+    finally:
+        selinux.selinux_status_close()
+    return mode == ENFORCING
```

Comment:
The python module is auto-generated from the C source-code, so yes the python docs are really really ugly.  The best thing is to read the C documentation...which I can't find again at the moment.